spec/sasl_ssl_spec.lua

```diff
@@ -56,3 +59,20 @@ describe("Testing sasl ssl client", function()
     assert.is_number(tonumber(offset))
   end)
 end)
+
+describe("Testing sasl plain client with tcp proxy", function()
+  before_each(function()
+    cli = client:new(broker_list_sasl_ssl_proxy, client_config_sasl_ssl)
```

Comment:
```suggestion
```

This is not wanted.

Reply:
I want to test here with the TCP proxy, which is why I create a new client that points to the proxy instead of the actual broker.